test/e2e/suite/005-app-wide-update-strategy/01-install.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: image-updater-e2e-005-01
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: image-updater-005-01
+  annotations:
+    argocd-image-updater.argoproj.io/image-list: guestbook=gcr.io/heptio-images/ks-guestbook-demo:~0
+    argocd-image-updater.argoproj.io/guestbook.update-strategy: semver
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/argoproj/argocd-example-apps.git
+    path: kustomize-guestbook
+    targetRevision: HEAD
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: image-updater-e2e-005-01
+  syncPolicy:
+    automated: {}
+    retry:
+      limit: 2
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: image-updater-e2e-005-02
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: image-updater-005-02
+  annotations:
+    argocd-image-updater.argoproj.io/image-list: guestbook=gcr.io/heptio-images/ks-guestbook-demo:latest
+    argocd-image-updater.argoproj.io/guestbook.update-strategy: latest
```

Comment:
this is still per-image update strategy. Since we're testing app wide update strategy, we should use `update-strategy:` instead of `guestbook.update-strategy:`